File: IE 211 Fall21/src/Lab1/Cat.java
package Lab1;

public class Cat extends Animal{
	
	private int age;
	
	public Cat(){}
	
	public void setAge(int ageIn)
	{
		age = ageIn;
	}
	
	public void isYoungerThan(int ageIn)
	{
		if(age < ageIn)
			System.out.println("The cat is younger than " + ageIn);
		else 
			System.out.println("The cat is older than " + ageIn);
	}

	public void speak()
	{
		System.out.println("Miyavv!");
	}
}

File: IE211 Fall22/src/Lab1/Cat.py
from Lab1.Animal import Animal

class Cat(Animal):

    __age = 0

    def __init__(self):
        pass

    def setAge(self, age):
        self.__age  = age

    def isYoungerThan(self, age):
        if self.__age < age:
            print(f"The cat is younger than {age}")
        else:
            print(f"The cat is older than {age}")

    def speak(self):
        print("Meow :)")